api/client/client.go

```diff
@@ -138,6 +157,41 @@ func unix2HTTP(u *url.URL) {
 	}
 }
 
+// shouldRoundTripRetry
+func (c *Client) shouldRoundTripRetry(res *http.Response, err error) bool {
+	if http.ErrHandlerTimeout == err || res == nil {
+		return true
+	}
+	return res.StatusCode == http.StatusRequestTimeout ||
+		res.StatusCode == http.StatusGatewayTimeout
+}
+
+// RoundTrip
```

Comment:
is this required for this change?

Reply:
> is this required for this change?

This change was on px master. Looks like openstorage master was not vendored into porx master and this change was mistakenly carried over.. I manually removed this change in a later commit. Thanks for the catch!